maxim/logger/parsers/generation_parser.py

```
@@ -93,6 +94,20 @@ def parse_generation_error(error_data):
     return error_data
 
 
+def default_json_serializer(o: Any) -> Any:
```

Ruff (0.11.9)

Dynamically typed expressions (typing.Any) are disallowed in `o`

(ANN401)